package.json

```diff
@@ -1,15 +1,15 @@
 {
     "name": "@dell/clarity-react",
-    "version": "0.24.6",
+    "version": "0.25.2",
```

Comment:
We already have package for "0.25.2" so need to increase version to "0.25.3"

Reply:
Done